docs/05-storage-providers.md

```diff
@@ -263,6 +263,7 @@ To guard against man-in-the-middle attacks, the server fingerprint is verified o
 * `--ssh-keyfile`
 Points to a valid OpenSSH keyfile. If the file is encrypted, the password supplied is used to decrypt the keyfile.
 If this option is supplied, the password is not used to authenticate. This option only works when using the managed SSH client.
+This key should be in PEM format, e.g. created with `ssh-keygen -m pem`.

```

Comment:
**thought:** I'm choosing more commital language than I would necessarily like. Something more relaxed and future-proof:

```suggestion
This key may need to be in PEM format, e.g. created with `ssh-keygen -m pem`, if Duplicati reports an error like _openssh key type: ssh-rsa is not supported_.

```